frontend/assets/javascripts/src/utils/form/Password.js

```diff
@@ -79,9 +70,9 @@ function ($, bean, component) {
 
         require(['js!zxcvbn'], function() {
 
-            var $passwordStrengthIndicatorTemplate = self.$passwordStrengthIndicatorTemplate();
-
-            $passwordStrengthIndicatorTemplate.insertAfter(self.getElem('PASSWORD_STRENGTH_INPUT'));
+            utilsHelper.toArray(document.querySelectorAll('.' + self.classes.STRENGTH_INDICATOR)).map(function (elem) {
+                elem.classList.toggle('is-hidden');
+            });
```

Comment:
Yup spoke with @mattandrews about this - agreed we don't want code looping through things all over the codebase, we could write our own methods to abstract this out but this is essentially re-inventing the wheel.
Lets plan a CS-dev chat about where we want to head with the JS and what we use from these libs and if we need them or want to look at other solutions. As @davidrapson we have enough native stuff to think about this.